tor/core/user_interaction.py

```diff
@@ -88,7 +88,10 @@ def process_claim(post, config):
 
         if flair.unclaimed in top_parent.link_flair_text:
             # need to get that "Summoned - Unclaimed" in there too
-            post.reply(_(claim_success))
+            if first_claim:
+                post.reply(_(claim_success.format(first_claim_success)))
```

Comment:
(might be a moot point with the other comments I left, but...)

Looks like we're missing an import for `first_claim_success`

Reply:
:+1:

Reply:
Still needs the missing import. Then I'll approve the PR.